sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/FunctionTableSubqueryArgumentExpression.scala

```diff
@@ -86,11 +88,20 @@ case class FunctionTableSubqueryArgumentExpression(
   override def hint: Option[HintInfo] = None
   override def withNewHint(hint: Option[HintInfo]): FunctionTableSubqueryArgumentExpression =
     copy()
+  override def withNewNestedOuterAttrs(
+    nestedOuterAttrs: Seq[Expression]
+  ): FunctionTableSubqueryArgumentExpression = {
+    assert(nestedOuterAttrs.toSet.subsetOf(outerAttrs.toSet),
+      s"nestedOuterAttrs must be a subset of outerAttrs, " +
+        s"but got ${nestedOuterAttrs.mkString(", ")}")
+    copy(nestedOuterAttrs = nestedOuterAttrs)
+  }
```

Comment:
```suggestion

  override def withNewNestedOuterAttrs(
    nestedOuterAttrs: Seq[Expression]
  ): FunctionTableSubqueryArgumentExpression = {
    assert(nestedOuterAttrs.toSet.subsetOf(outerAttrs.toSet),
      s"nestedOuterAttrs must be a subset of outerAttrs, " +
        s"but got ${nestedOuterAttrs.mkString(", ")}")
    copy(nestedOuterAttrs = nestedOuterAttrs)
  }

```
Please add new lines